src/main/java/frc/robot/RobotContainer.java

```diff
@@ -71,6 +88,7 @@ public RobotContainer() {
    */
   private void configureBindings() {
     xButton.whileTrue(gripperSystem.intake());
+   
```

Comment:
i thought this pr was supposed to implement button bindings?

Reply:
No